spec/requests/comments/create_spec.rb

```diff
@@ -0,0 +1,109 @@
+require 'swagger_helper'
+include JsonResponseHelpers
+include AttachmentUrlHelper
+
+RSpec.describe 'comments', type: :request, capture_examples: true do
+  let!(:user) { create :user }
+  let!(:project) { create :project, user: user }
+  let!(:task) { create :task, project: project }
+  let!(:not_allowed_task) { create :task }
+  let!(:comment) { build :comment, :with_jpg_attachment, task: task }
+
+  def token
+    Knock::AuthToken.new(payload: { sub: user.id }).token
+  end
+
+  path '/api/v1/tasks/{task_id}/comments' do
+    post(summary: 'create new comment') do
+      tags 'comments'
+      parameter 'Content-Type', in: :header, type: :string
+      parameter 'task_id', in: :path, type: :string
+      parameter 'Authorization', in: :header, type: :string
+      parameter 'body', in: :body, type: :formData, schema: { '$ref' => '#/definitions/comment' }
+      let(:'Content-Type') { 'application/json' }
+      response(201, description: 'successful') do
+        let(:task_id) { task.id }
+        let(:Authorization) { "Bearer #{token}" }
+        let(:body) do
+          { data: {
+            attributes: {
+              text: comment.text,
+              attachment: blob_for('photo.jpg').signed_id
+            }
+          } }
+        end
+        around do |example|
+          expect { example.run }.to change(Comment, :count).by(1)
```

Comment:
`expect { example.run }.to change(Comment, :count).by(1).and change(ActiveStorage::Attachment, :count).by(1)`
You can chain change expectations.